src/data-types/mailstream_cfstream.h

```diff
@@ -62,6 +62,9 @@ extern "C" {
     MAILSTREAM_CFSTREAM_SSL_LEVEL_SSLv2,
     MAILSTREAM_CFSTREAM_SSL_LEVEL_SSLv3,
     MAILSTREAM_CFSTREAM_SSL_LEVEL_TLSv1,
+    MAILSTREAM_CFSTREAM_SSL_LEVEL_TLSv1_1,
```

Comment:
Could you move them to the end so that the value of the constants won't change?

Reply:
Done.